bedrock/settings/base.py

```diff
@@ -1346,9 +1344,6 @@ def lazy_email_backend():
 CSP_SCRIPT_SRC = CSP_DEFAULT_SRC + (
     # TODO fix things so that we don't need this
     "'unsafe-inline'",
-    # TODO snap.svg.js passes a string to Function() which is
-    # blocked without unsafe-eval. Find a way to remove that.
-    "'unsafe-eval'",
```

Comment:
@pmac this PR removes `snap.svg.js` from the repo. I guess we are now safe to remove `unsafe-eval` here? :)

Reply:
Ooooooo. Yeah! Hopefully so! Lets try that in another pr. Good catch 🔥

Reply:
Ok, will leave this in for now